### PR TITLE
Update docs with grant access

### PR DESCRIPTION
**Pull request by sarah-clements**
Sebastian had grants for the replica access for the auth_user table as `GRANT SELECT (id, username, email) ON treeherder.auth_user to 'myuser';` per our docs, but I noticed that my user access was `GRANT SELECT, SELECT (id, username, email) ON treeherder.auth_user to 'myuser';` and updating his access to what mine was solved the issue he was hitting.

But I'm not as well-versed in SQL and its not clear to me from looking at the mysql docs why the latter worked but the former didn't. @armenzg or @klahnakoski do you have some insight into this?

### CODE REVIEW

File: docs/infrastructure/administration.md [hunk 1]
```diff
@@ -312,14 +312,16 @@ CREATE USER 'myuser' IDENTIFIED BY 'PASSWORD >=30 CHARACTERS LONG' REQUIRE SSL;
 -- Tables where we want to allow only partial access.
 -- `password` is randomly generated by Django and never used/exposed due to SSO,
 -- so is not really sensitive, but it causes less confusion to still exclude it.
-GRANT SELECT (id, username, email) ON treeherder.auth_user to 'myuser';
+GRANT SELECT, SELECT (id, username, email) ON treeherder.auth_user to 'myuser';
```
Comment (armenzg) [inline docs/infrastructure/administration.md:315]: This is really weird! I'm not as familiar with MySql so I don't know why that double SELECT would make sense.

Comment (klahnakoski) [reply docs/infrastructure/administration.md:315]: I *think* this is a `GRANT` command with two, comma separated, values.  The first allows the user to perform SELECT queries (the ability to consume server resources, with nothing said about what tables), the second is allowing specific table/columns.   This command is not SQL.

Comment (sarah-clements) [reply docs/infrastructure/administration.md:315]: So... is it something specific to MySQL?

Comment (klahnakoski) [reply docs/infrastructure/administration.md:315]: Yes, it is specific to the MySQL permissions system.  Although the other database vendors have something **very** similar; with differences in what resources are available.  Maybe it has been standardized?

Comment (sarah-clements) [reply docs/infrastructure/administration.md:315]: Ok, well either way let's get it added to our docs.

**Pull request merged**